### ISSUE TITLE
Still an error after yls8.mtheall.com is back!

### ISSUE THREAD

**Issue by IgorEisberg** [yellows8/3ds_homemenuhax#233]
It's odd that there's still an error code 0xd8a0a028 even when the website is accessible.
[menuhax_manager.log.txt](https://github.com/yellows8/3ds_homemenuhax/files/617676/menuhax_manager.log.txt)

**Comment by 0psychonaut0** [comment]
I'm getting the exact same problem...
Maybe something is wrong with the server?

**Comment by yellows8** [comment]
Fixed.

**Comment by AgustinLado** [comment]
Hello! I've got the same problem, and yls8.mtheall.com is definitely up. I'm unsure whether to open a new issue or continue this one, so here it goes.

I'm using a **New 3DS XL** at version **10.7.0-32U** and entering via browserhax.
I've tried creating a config file in the SD root (as well as /menuhax/ just in case) with the config found [here](http://pastebin.com/952s88vg) as well as the ropbin for my current version.

I didn't want to ask, but I've tried this several times in several ways and the error is consistent. Thanks!
